lib/resources/example_json/r4_examples_medication_request.rb

```diff
@@ -40,6 +40,14 @@ module Resources
             "ingredient": [
               {
                 "itemCodeableConcept": {
+                  "coding": [
+                    {
+                      "system": "https://fhir.cerner.com/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/synonym",
```

Comment:
These examples now should point to new sandbox tenant, ref: https://github.com/cerner/fhir.cerner.com/commit/bc0d9d61416173f030bc6af11e263a4830fb1c3f

Reply:
Addressed